ansible/README.md

```diff
@@ -0,0 +1,27 @@
+This directory contains an Ansible host role for VAST.
+The role can be parameterized with three variables:
```

Comment:
Nit: All throughout the Markdown files, can you reflow text to 80 columns as per our style guide?